extension/textext/asktext.py

```diff
@@ -718,7 +718,7 @@ def cb_key_press(self, widget, event, data=None):
 
             # Cancel dialog via shortcut if set by the user
             close_shortcut_value = self._gui_config.get("close_shortcut", self.DEFAULT_CLOSE_SHORTCUT)
-            if close_shortcut_value is not 'None':
+            if close_shortcut_value != 'None':
```

Comment:
This check can be removed completely, right?

Reply:
...indeed :smiley: